src/history/test/HistoryTests.cpp

```diff
@@ -918,93 +918,149 @@ TEST_CASE("History catchup", "[history][catchup][acceptance]")
         // generate megabytes of unnecessary log entries
         CatchupSimulation catchupSimulation{VirtualClock::REAL_TIME};
         auto checkpointLedger = catchupSimulation.getLastCheckpointLedger(3);
-        auto app = catchupSimulation.createCatchupApplication(
-            std::numeric_limits<uint32_t>::max(),
-            Config::TESTDB_BUCKET_DB_PERSISTENT, "app");
 
         auto offlineNonCheckpointDestinationLedger =
-            checkpointLedger -
-            HistoryManager::getCheckpointFrequency(app->getConfig()) / 2;
+            checkpointLedger - HistoryManager::getCheckpointFrequency(
+                                   catchupSimulation.getApp().getConfig()) /
+                                   2;
 
-        SECTION("when not enough publishes has been performed")
+        INFO("when not enough publishes has been performed")
```

Comment:
nit: Please move the INFOs *inside* the scopes. The way this works is that catch instantiates some object that marks the info line lifetime, and then when the object is removed (gone out of scope) it won't log that line anymore. So with the current implementation the log for a failure inside 'offline, in the middle of checkpoint' section would look like 
```
when not enough publishes has been performed
online
offline
offline, in the middle of checkpoint
```
While if you move the INFOs inside scope you would get
```
when not enough publishes has been performed
offline, in the middle of checkpoint
```
which is pretty much the same as what we used to get with the sections.